src/System.Runtime.Extensions/src/System/IO/TextWriter.cs

```diff
@@ -109,7 +109,9 @@ public virtual string NewLine
         // but descendant classes can override the method to provide the
         // appropriate functionality.
         //
-        public abstract void Write(char value);
+        public virtual void Write(char value)
+        {
+        }
```

Comment:
It's wacky. However, it's like that back to at least 2.0, and apparently intentionally so according to the comment there.
```
        // Writes a character to the text stream. This default method is empty,
        // but descendant classes can override the method to provide the
        // appropriate functionality.
```

If we don't fix this (ie if we don't take this PR), perfectly good implementations of TextWriter that have worked for a long time on desktop (presumably using Write(string)) could fail to instantiate on Core. Even if we back-port to desktop, that just introduces the same break between desktop versions.

LMK which you prefer.

If we do fix it, we introduce the risk that someone implements TextWriter but forgets to implement Write(char). That seems a lesser concern.